lua/entities/gmod_tardis/modules/sh_teleport.lua

```diff
@@ -8,7 +8,11 @@ TARDIS:AddKeyBind("teleport-demat",{
 		if TARDIS:HUDScreenOpen(ply) then return end
 		local pilot = self:GetData("pilot")
 		if SERVER then
-			if ply==pilot and (not down) then
+			if ply==pilot and down then
+				ply:SetTardisData("teleport-demat-bind-down", true)
+			end
+			if ply==pilot and (not down) and ply:GetTardisData("teleport-demat-bind-down") then
+				ply:SetTardisData("teleport-demat-bind-down", false)
```

Comment:
Always set to nil if `ply:GetTardisData("teleport-demat-bind-down", false) and down == false` regardless of pilot status